src/serac/physics/utilities/functional/tensor.hpp

```diff
@@ -376,8 +375,9 @@ SERAC_HOST_DEVICE constexpr auto operator-(const tensor<T, n...>& A)
 template <typename S, typename T, int... n>
 SERAC_HOST_DEVICE constexpr auto operator-(const tensor<S, n...>& A, const tensor<T, n...>& B)
 {
+  // FIX: hcurl_unit_tests
```

Comment:
what does this refer to?

Reply:
Sorry. I think it was related to some of the type deduction nvcc issues.